pydantic_core/core_schema.py

```diff
@@ -670,6 +1128,27 @@ class FunctionWrapSchema(TypedDict, total=False):
 def function_wrap_schema(
     function: WrapValidatorFunction, schema: CoreSchema, *, ref: str | None = None, extra: Any = None
 ) -> FunctionWrapSchema:
+    """
+    Returns a schema that wraps a function around the result of schema validation, e.g.:
+
+    ```py
+    from pydantic_core import SchemaValidator, core_schema
+
+    def fn(v: str, **kwargs) -> str:
```

Comment:
This is wrong, you'll probably want to use an example like [this](https://github.com/pydantic/pydantic-core/blob/96939192a1386882f720c13c3c99cf20d14140f9/tests/validators/test_function.py#L94-L96).

Reply:
Sounds good - I used that example